include/boost/certify/detail/keystore_windows.ipp

```diff
@@ -96,7 +96,7 @@ BOOST_CERTIFY_DECL std::unique_ptr<::CERT_CONTEXT const, cert_context_deleter>
     if (!CertAddEncodedCertificateToStore(store.get(),
                                           X509_ASN_ENCODING,
                                           buffer.data(),
-                                          buffer.size(),
+                                          static_cast<DWORD>(buffer.size()),
```

Comment:
Should probably fastfail if buffer.size() > DWORD_MAX?

Reply:
Hi, thanks for your feedback and sorry for the late answer.
The buffer is filled in dump_cert by the openssl function i2d_X509. The size is set to the return value of this function, which is an int. So, technically, buffer.size() cannot be larger than DWORD_MAX.
Would you agree that this is good enough? Should i add comments explaining this?

Reply:
Makes sense to me. I would add a comment.